roles/openshift_metering/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+openshift_metering_operator_image: ''
```

Comment:
For images, we've moved to a pattern that is based off the value of `l_os_registry_url`.
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/defaults/main.yml#L14

Here is the commit that introduced that for reference: https://github.com/openshift/openshift-ansible/commit/a310ad860090d050b9c71a54ea0b1cc38f104db4

Reply:
That doesn't apply to this component yet, as we're still using images published on quay.io. We're still figuring out where the images will live, and how we publish for longer-term. This is still alpha quality.